yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java

```diff
@@ -9,55 +9,51 @@
 public class YotiClientProperties {
 
     /**
-     * The Application ID provided by Yoti dashboard.
+     * The SDK client ID provided by Yoti dashboard.
      */
-    private String applicationId;
+    private String clientSdkId;
 
     /**
-     * The SDK client ID provided by Yoti dashboard.
+     * The scenario ID provided by Yoti Hub
```

Comment:
Since this change (and the change from dashboard to hub on line 12) have already been updated in https://github.com/getyoti/yoti-java-sdk/pull/113, it is probably best to not change them again here too, as you're more likely to have a merge conflict
```suggestion
     * The scenario ID provided by Yoti dashboard
```